bin/check-marathon-task.rb

```diff
@@ -71,7 +71,8 @@ class MarathonTaskCheck < Sensu::Plugin::Check::CLI
   option :instances,
          short: '-i INSTANCES',
          long: '--instances INSTANCES',
-         required: true,
+         required: false,
+         default: 0,
```

Comment:
I could be missing something but this does not seem like a good default. If we change it I feel like it should at least be 1. Can you explain the rationale?

Reply:
I think I understand this now, without specifying any it will pull from marathon what it thinks it should be. Does this behavior exist in all versions of marathon or was this a recent feature added?

Reply:
yes, this exists on all versions of marathon, before you would have to specify an instance count and the plugin it would crash if you set it to 0, now, if you set it to 0 or if you omit it, the plugin will check the instance count that marathon is expecting, I think this is a sane default.